packages/forms/demo/stories/FileStory.tsx

```diff
@@ -10,10 +10,12 @@ import { Story } from '@storybook/react';
 import { Progress } from '@zendeskgarden/react-loaders';
 import { File, IFileProps } from '@zendeskgarden/react-forms';
 
-interface IArgs extends IFileProps {
+interface IArgs extends Omit<IFileProps, 'onClick'> {
   hasClose?: boolean;
   hasDelete?: boolean;
   value?: number;
+  onClick?: (event: React.MouseEvent<HTMLButtonElement>) => void;
+  onCloseKeydown?: (event: React.KeyboardEvent<HTMLButtonElement>) => void;
```

Comment:
```suggestion
  onCloseKeydown?: ButtonHTMLAttributes<HTMLButtonElement>['onKeyDown'];
```

Garden tends to prefer making direct association with the component type, when possible.